src/LayoutManager/LayoutStrategies/FixedLayout.ts

```diff
@@ -16,14 +17,16 @@ export class FixedLayout extends LayoutStrategy {
     const result = super.calcBoundingBox(objects, context);
     if (context.type === 'initialization' && result) {
       const {
-        target: { width, height },
+        target: { width, height, originX, originY },
       } = context;
+      const sizeCorrection = new Point(width, height)
+        .multiply(new Point(-resolveOrigin(originX), -resolveOrigin(originY)))
+        .transform(context.target.calcOwnMatrix(), true);
 
       return {
         ...result,
-        center: result.center.add(
-          new Point(width, height).subtract(result.size).scalarDivide(2)
-        ),
+        // center: result.center.add(sizeCorrection),
+        // correction: new Point(result.correction).add(sizeCorrection),
```

Comment:
I am struggling with supporting a fixed size

Reply:
But this is a newly discovered bug or something that came out because of changes requested?